modules/statics/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 # Each line is a file pattern followed by one or more owners.
 
 # These owners and first officers will be the default owners for everything in the repo.
-*   tyler@bitgo.com
+*   @BitGo/sdk-team
```

Comment:
I think we can remove this file - only the root codeowners file is actually used